scrnaseq.Rmd

```diff
@@ -1713,14 +1710,14 @@ if (markers_found) {
 ```{r markers_plot_violin_raw, eval=markers_found, fig.height=fig_height_markers}
 if (markers_found) {
   # Violin plot of raw gene expression counts
-  p_list = Seurat::VlnPlot(sc, features=markers_examples, assay="RNA", slot="counts", combine=FALSE, pt.size=0.2)
+  p_list = Seurat::VlnPlot(sc, features=markers_examples, assay="RNA", slot="counts", combine=FALSE, pt.size=0)
```

Comment:
I like to see the dots, and prefer if we leave them in small-ish like before.

Reply:
ok

Reply:
though you do not get much information from the dots and they potentially obscur the violin

Reply:
Still, pls leave them for now.